build-tools-internal/src/main/groovy/elasticsearch.build-complete.gradle

```diff
@@ -8,86 +8,10 @@
-        fileset(dir: projectDir) {
-          Set<File> fileSet = fileTree(projectDir) {
-            include("**/*.hprof")
-            include("**/build/test-results/**/*.xml")
-            include("**/build/testclusters/**")
-            include("**/build/testrun/*/temp/**")
-            include("**/build/**/hs_err_pid*.log")
-            exclude("**/build/testclusters/**/data/**")
-            exclude("**/build/testclusters/**/distro/**")
-            exclude("**/build/testclusters/**/repo/**")
-            exclude("**/build/testclusters/**/extract/**")
-            exclude("**/build/testclusters/**/tmp/**")
-            exclude("**/build/testrun/*/temp/**/data/**")
-            exclude("**/build/testrun/*/temp/**/distro/**")
-            exclude("**/build/testrun/*/temp/**/repo/**")
-            exclude("**/build/testrun/*/temp/**/extract/**")
-            exclude("**/build/testrun/*/temp/**/tmp/**")
-          }
-            .files
-            .findAll { Files.isRegularFile(it.toPath()) }
-
-          if (fileSet.empty) {
-            // In cases where we don't match any workspace files, exclude everything
-            ant.exclude(name: "**/*")
-          } else {
-            fileSet.each {
-              ant.include(name: projectDir.toPath().relativize(it.toPath()))
-            }
-          }
-        }
-
-        fileset(dir: "${gradle.gradleUserHomeDir}/daemon/${gradle.gradleVersion}", followsymlinks: false) {
-          include(name: "**/daemon-${ProcessHandle.current().pid()}*.log")
-        }
-
-        fileset(dir: "${gradle.gradleUserHomeDir}/workers", followsymlinks: false)
-
-        fileset(dir: "${project.projectDir}/.gradle/reaper", followsymlinks: false, erroronmissingdir: false)
-      }
-    } catch (Exception e) {
-      logger.lifecycle("Failed to archive additional logs", e)
-    }
-
-    if (uploadFile.exists() && System.getenv("BUILDKITE") == "true") {
-      try {
-        println "Uploading buildkite artifact: ${uploadFilePath}..."
-        new ProcessBuilder("buildkite-agent", "artifact", "upload", uploadFilePath)
-          .start()
-          .waitFor()
-
-        println "Generating buildscan link for artifact..."
-
-        def process = new ProcessBuilder("buildkite-agent", "artifact", "search", uploadFilePath, "--step", System.getenv('BUILDKITE_JOB_ID'), "--format", "%i").start()
-        process.waitFor()
-        def artifactUuid = (process.text ?: "").trim()
-
-        println "Artifact UUID: ${artifactUuid}"
-        if (artifactUuid) {
-          buildScan.link 'Artifact Upload', "https://buildkite.com/organizations/elastic/pipelines/${System.getenv('BUILDKITE_PIPELINE_SLUG')}/builds/${buildNumber}/jobs/${System.getenv('BUILDKITE_JOB_ID')}/artifacts/${artifactUuid}"
-        }
-      } catch (Exception e) {
-        logger.lifecycle("Failed to upload buildkite artifact", e)
-      }
-    }
-  }
-}
+project.getPlugins().apply(ElasticsearchBuildFinishedPlugin.class)
```

Comment:
Can we just apply this plugin to the root project and ditch this pre-compiled script plugin altogether?

Reply:
fixed